content/docs/howto/java.md

```diff
@@ -386,18 +394,11 @@ docker run --rm --tty \
 
 ## Enable Remote Debugging
 
-If `BP_DEBUG_ENABLED` is set at build-time and `BPL_DEBUG_ENABLED` is set at runtime the [Debug Buildpack][bp/debug] will configure the application to accept debugger connections. The debug port defaults to `8000` and can be configured with `BPL_DEBUG_PORT` at runtime. If `BPL_DEBUG_SUSPEND` is set at runtime, the JVM will suspend execution until a debugger has attached.
+If `BPL_DEBUG_ENABLED` is set to `true` at runtime the application will be configured to accept debugger connections. The debug port defaults to `8000` and can be configured with `BPL_DEBUG_PORT` at runtime. If `BPL_DEBUG_SUSPEND` is set at runtime, the JVM will suspend execution until a debugger has attached.
 
 **Example**: Remote Debugging
 
-The following commands builds a debug-enabled image.
-{{< code/copyable >}}
-pack build samples/java \
-  --path java/jar \
-  --env BP_DEBUG_ENABLED=true
-{{< /code/copyable >}}
-
-To run the image with the debug port published:
+To run an image with the debug port published:
```

Comment:
```suggestion
The following command builds a debug-enabled image.

{{< code/copyable >}}
pack build samples/java \
  --path java/jar
{{< /code/copyable >}}

To run an image with the debug port published:
```

I think we should still leave in the build command even though there's nothing specific that needs to be done. It just makes the how-to feel more complete.